grammars/css.cson

```diff
@@ -1913,7 +1913,8 @@
             'match': '''(?x)
               (-?(?!\\d)(?:[\\w-]|[^\\x00-\\x7F]|\\\\(?:[0-9a-fA-F]{1,6}|.))+)
               \\s*
-              (?=[~|^\\]$*=]|/\\*)
```

Comment:
@nathansobo I'll wait for you (or somebody else) to confirm this before attempting a different solution. Trying to tie up loose ends elsewhere ATM.